packages/liveblocks-react-comments/src/primitives/Timestamp.tsx

```diff
@@ -80,12 +80,38 @@ function formatShortDate(date: Date, locale?: string) {
   return capitalize(formatter.format(date));
 }
 
+// Some locales' relative formatting can be broken (e.g. "-1h") when using the narrow style.
+const localesWithBrokenNarrowRelativeFormatting = [
```

Comment:
It feels strange to do this instead of using `short` everywhere! 

But I trust there's a reason behind using `narrow` where we can, and that you've already considered the options 👍.

Reply:
I think it's just because those locales don't support `narrow` correctly (yet), from the browsers' side. The alternative to cherry-picking these locales like I did (using xhqmwd.csb.app) is to use `short` for all locales but it's a bit unfortunate to give up on "1h ago" (and go back to "1 hour ago") just because of a few locales imo.